odoo/cli/upgrade_code.py

```diff
@@ -108,6 +108,8 @@ def migrate(
 
 
 class UpgradeCode(Command):
+    """ Activates code migration scripts between the chosen versions """
```

Comment:
https://github.com/odoo/odoo/pull/185387/files#diff-527cbc0a9ff314cc12aafc04d0da70dff607f5cdc6a8826a7491e3e295d53620
```suggestion
    """ Mass-rewrite the entire source code using the scripts found at /odoo/upgrade_code """
```

Reply:
Sounds a bit aggressive. I know it's on purpose...
But can I remove "Mass-" and keep "entire source code" ?